### ISSUE TITLE
false no-member: Module 'numpy' has no 'degrees' member

### ISSUE THREAD

**Issue by belm0** [PyCQA/pylint#3856]
### Steps to reproduce

```python
import numpy
numpy.degrees(1)
```

### pylint --version output
```
pylint 2.6.0
astroid 2.4.2
Python 3.7.6 (default, Dec 30 2019, 19:38:36)
[Clang 10.0.0 (clang-1000.11.45.5)]
```

numpy version: 1.16.4

**Comment by belm0** [comment]
also seen on same pylint/astroid version, after upgrading to numpy 1.19.2:

```
E1101: Module 'numpy.random' has no 'random' member (no-member)
```

**Comment by hippo91** [comment]
@belm0 thanks for the report.

**Comment by hippo91** [comment]
@belmo it's worth noting that starting with version `1.18`, due to a modification in `numpy` and regarding the actual code inside `pylint`, no `no-member` message will be emitted anymore for `numpy`'s attribute.
